stache-boilerplate/.travis.yml-sample

```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+before_script:
+  - npm i -g grunt-cli blackbaud-stache-cli
+
+script:
+  - stache build --config=stache.yml,stache.prod.yml,stache.deploy.yml
```

Comment:
With this script update, will we still be able to build locally? Or is the only build process now through github/Travis? All of the User Ed how-to pages are currently manually deployed to the bbweb server. These are the only projects I know of that deploy this way though.